yt_dlp/extractor/twitter.py

```diff
@@ -552,10 +701,84 @@ class TwitterIE(TwitterBaseIE):
         'only_matching': True,
     }]
 
+    def _graphql_to_legacy(self, data, twid):
+        def filter_tweet(_, obj):
+            return obj['entryId'] == f'tweet-{twid}'
+
+        path = ('threaded_conversation_with_injections_v2', 'instructions', 0, 'entries',
+                filter_tweet, 'content', 'itemContent', 'tweet_results', 'result')
+        result = traverse_obj(data, path, expected_type=dict, default={}, get_all=False)
+
+        if 'tombstone' in result:
+            cause = traverse_obj(result, ('tombstone', 'text', 'text'), expected_type=str, default='')
+            raise ExtractorError(f'Twitter API says: {cause}', expected=True)
+
+        status = result.get('legacy', {})
+        status.update(traverse_obj(result, {
+            'user': ('core', 'user_results', 'result', 'legacy'),
+            'card': ('card', 'legacy'),
+            'quoted_status': ('quoted_status_result', 'result', 'legacy'),
+        }, expected_type=dict, default={}))
+
+        # extra transformations are needed bc LeGaCy cOmPaTiBiLiTy
```

Comment:
?

Reply:
Eventhough the fields are named `legacy` it does not conform to the legacy format. Thats where this comment stems from. I will clarify it.